Sources/Apollo/ApolloStore.swift

```diff
@@ -241,6 +241,10 @@ public final class ApolloStore {
       try body(&object)
       try write(object: object, withKey: key, variables: variables)
     }
+    
+    public func removeRecord(for key: CacheKey) throws {
```

Comment:
For consistency with the other methods, I would make this `removeObject(withKey:)`. (I don't think we should use record as part of the public API in general, only for cache implementors.)

Reply:
Fair